app/Http/Controllers/TaskController.php

```diff
@@ -158,4 +158,17 @@ public function showResource(Request $request) : TodoResourceCollection
         $tasks = $this->taskService->all();
         return new TodoResourceCollection($tasks);
     }
+    public function archived()
```

Comment:
This is not how to go about doing this. We are not using eloquent or anything like that
so there's no static where method you can call on the task

Reply:
I’m not the one that did todo resourcecollection o

Reply:
Can I use soft delete?